src/evaluators/Program.js

```diff
@@ -251,4 +258,55 @@ export default function(ast: BabelNodeProgram, strictCode: boolean, env: Lexical
 
   invariant(val === undefined || val instanceof Value);
   return val || realm.intrinsics.empty;
+
+  function emitThrow(value: Value) {
```

Comment:
Can these `emit...` functions get moved to the Generator? Seems more consistent to me.

Reply:
Will do when I have more time and look into exception handling for require calls.